app/services/process_submission_service.rb

```diff
@@ -26,22 +39,22 @@ def perform
   def send_email(mail)
     if number_of_attachments(mail) <= 1
       response = EmailService.send_mail(
-          from: mail.from,
-          to: mail.to,
-          subject: mail.subject,
-          body_parts: retrieve_mail_body_parts(mail),
-          attachments: attachments(mail)
+        from: mail.from,
+        to: mail.to,
+        subject: mail.subject,
+        body_parts: retrieve_mail_body_parts(mail),
+        attachments: attachments(mail)
       )
 
       submission.responses << response.to_h
```

Comment:
We ok to not return responses at this point?
If so, let's proceed without them for now

Reply:
this is existing email code not new code (it has been linted)